### DISCUSSION TITLE
Does RMS ID in our code refer to the unique IDs of Items?

### DISCUSSION THREAD

**Discussion by johnnyhchen** [gp-seattle/rms-app#164] [Backend Questions]
In ts-code/src/db/ItemsTable.ts we encountered an Error message we would throw that says 
```
    /**
     * Adds reservation to table
     */
    public create(
        id: string,
        reserver: string,
        itemId: string,
        startTime: string,
        endTime: string,
    ): Promise<DocumentClient.PutItemOutput> {
        return this.get(id)
            .then((entry: ScheduleSchema) => {
                if (entry) {
                    throw Error(`RMS ID ${id} is not unique.`)

```
We are copying this code for the ScheduleTable and was wondering if the RMS ID refers uniquely to the IDs of rows in the Items table (so we can call the ID for the reservation table something else, like "Reservation ID is not unique" in our error message).

**Comment by yauj** [comment]
id here is unique to the table. So it's id of the item table here.

You would want `id` of the history table.
